### DISCUSSION TITLE
Gemini-1.5 Pro and gemini advanced not available in the app

### DISCUSSION THREAD

**Discussion by kneelesh48** [danny-avila/LibreChat#2191] [Feature Requests & Suggestions]
Gemini 1.5 pro and gemini advanced are not available. Can I just edit the env file to add them? Doesn't it automatically fetch list of available models from the API?

**Comment by danny-avila** [comment]
Will add it soon!

**Comment by danny-avila** [comment]
Gemini 1.5 is now supported: https://github.com/danny-avila/LibreChat/pull/2383

**Reply by Dual-0** [reply]
I go this in the logs:
```
call to gauth.request message: [{
  "error": {
    "code": 400,
    "message": "Invalid Endpoint name: projects/gemini-xxx/locations/us-central1/publishers/google/models/gemini-1.5-pro-preview-0409 .",
    "errors": [
      {
        "message": "Invalid Endpoint name: projects/gemini-xxx/locations/us-central1/publishers/google/models/gemini-1.5-pro-preview-0409 .",
        "domain": "global",
        "reason": "badRequest"
      }
    ],
    "status": "INVALID_ARGUMENT"
  }
}
]
```
I comment out `GOOGLE_KEY` and add the `auth.json`
What do I wrong?

**Reply by danny-avila** [reply]
@Dual-0 it looks like you added spaces after the model name

**Reply by Dual-0** [reply]
Ohh you are right. My fault. Thx!

**Comment by isaactallack** [comment]
Gemini 1.5 is working great, thanks!

How am I able to tell whether the endpoint being used is Vertex or AI Studio for a specific model? E.g. I know that Vertex is being used when using gemini-1.5-pro-preview-0409 but if I select gemini-1.0-pro - is this using AI Studio or Vertex?

**Reply by danny-avila** [reply]
You can’t use both services at the same time, it’s either/or. See the docs on the Google setup, which list which models are available on each service

**Comment by yosun** [comment]
i notice it is still listed as gemini pro - is that automatically gemini1.5 flash/pro latest?
